labs/dxp/7.x/installation-and-upgrades/02-setting-up-liferay-dxp/installing-and-managing-apps/managing-apps.md

```diff
@@ -0,0 +1,58 @@
+# Managing Apps
+
+The App Manager is a Control Panel screen where you can install and uninstall apps, and examine installed apps, modules, and OSGi components.
+
+Access the App Manager by selecting *Control Panel* &rarr; *Apps* &rarr; *App Manager*. The App Manager lists installed apps and modules and provides app upload functionality.
+
+## The App Listing
+
+The *Filter and Order* menu filters items and orders them by category, status, or title. Click the up or down arrows to sort items in ascending or descending order, respectively. To search for an app or module, use the search bar. This is often the quickest way to find items.
+
+![The App Manager manages apps, modules, and components installed in your DXP instance.](./managing-apps/images/01.png)
+
+Each item listed in the table contains the item's description, version, and status. Here are the statuses:
+
+* *Installed:* The item is installed to DXP.
+* *Resolved:* The item's dependencies are active. Resolved items can typically be activated. Some items, however, can't be  activated and are intended to remain in the Resolved state (e.g., WSDD modules containing SOAP web services).
+* *Active:* The item is running on DXP.
+
+Clicking each item's Actions button (![Actions](./managing-apps/images/02.png)) brings up a menu in which you can activate, deactivate, or uninstall that item.
+
+To view an item's contents, click its name in the table.
+
+* If you click an app, the app's modules are listed.
+* If you click a module, the module's OSGi components and portlets are listed.
+
+The component/portlet level is the farthest you can go without getting into the source code. At any level in the App Manager, there's a link trail above the listing that shows the item's hierarchical context.
+
+## Installing Apps
+
+The App Manager is a convenient way to install a [downloaded app](./downloading-apps.md) from your local machine to a local or remote DXP server.
+
+```important::
+   For installing and uninstalling apps in production-grade environments, install apps [using server startup](./installing-apps-in-production.md) instead of using the App Manager.
+```
+
+Here are steps for using the App Manager to install an app:
+
+1. Select *Upload* from the options button (![Options](./managing-apps/images/03.png)). The Upload dialog appears.
+
+1. Browse to and select the app on your machine
+
+1. Click *Install*.
+
+The app installs via [auto deploy](./app-installation-strategies.md#auto-deployment). When the install completes, close the dialog and you're ready to roll!
+
+## Disabling Apps
+
+To disable an installed app, use an app blacklist to do so. This will mark the app as unavailable, so it is no longer used. See [Blacklisting Apps](./blacklisting-apps.md) for more information.
```

Comment:
We should mention that the uninstall feature is broken and that blacklisting is the workaround. Here's one way we could write this section.
| Uninstalling an app deactivates it (if it's active) and makes it unavailable for activation. 
| 
| **Note:** The App Manager's uninstall, re-install, and re-activation features have issues (see [LPS-102506](https://issues.liferay.com/browse/LPS-102506)). As a workaround, [Blacklist apps](./blacklisting-apps.md) to disable them.

Reply:
@Alec-Shay The admonition will be easy to spot and remove once the issue is fixed. Otherwise, any workaround text will be easy to miss and we might not see that we have to remove/replace it.

Reply:
@jhinkey There seems a clear disagreement from what JR and I had concluded when we discussed this. I think we may need to all discuss this so we're not just going back-and-forth on this one (maybe in chat, if that's easiest).

Reply:
@Alec-Shay Yeah, let's get the three of us on a chat about this (if not by 5:15p ET today, then tomorrow). thanks.